nh_eobs/sql_statements.py

```diff
@@ -459,10 +463,8 @@ def get_collect_activities_sql(self, activity_ids_sql):
                 when ews1.id is not null and ews2.id is null then 'first'
                 when ews1.id is null and ews2.id is not null then 'no latest'
             end as ews_trend,
-            case
-                when ews0.frequency is not null then ews0.frequency
-                else 0
-            end as frequency,
+            coalesce(ews0.frequency, 0) as frequency,
+            coalesce(bg0.frequency, 0) as bg_frequency,
```

Comment:
why is coalesce used here? This suggests there's more than 1 row returned by ews0 which could be problematic

Reply:
i'm an idiot, misread coalesce as concatenate